docs/sample_config.yaml

```diff
@@ -862,6 +864,14 @@ log_config: "CONFDIR/SERVERNAME.log.config"
 #rc_3pid_validation:
 #  per_second: 0.003
 #  burst_count: 5
+#
+#rc_invites:
+#  per_room:
+#    per_second: 0.3
+#    burst_count: 10
+#  per_user:
+#    per_second: 0.003
+#    burst_count: 5
```

Comment:
if this is a ratelimit per recipient, does that mean that a spammer can DoS a given recipient so that they can't receive any invites from genuine users any more? what happens to genuine invites - are they just lost somehow?

Reply:
(could this sort of info be added to the comments in the config file, please?)

Reply:
Hmm, true. This does mean In some ways this is somewhat by design, the idea is that you don't want to have users inundated by invites from a lot of different fake accounts. The invites will see the 429 though, so at least its a visible thing. I guess we could ratelimit by `(sending_server, target_user)` for inbound remote invites so that a spammer would have to have accounts on different servers?

Reply:
maybe. Honestly, I'm kinda questioning how useful this limit is as it stands.